app/src/main/java/com/asfoundation/wallet/ui/iab/PaymentMethodsInteract.kt

```diff
@@ -0,0 +1,86 @@
+package com.asfoundation.wallet.ui.iab
+
+import android.util.Pair
+import com.appcoins.wallet.bdsbilling.WalletService
+import com.appcoins.wallet.gamification.repository.ForecastBonusAndLevel
+import com.asfoundation.wallet.entity.Balance
+import com.asfoundation.wallet.entity.TransactionBuilder
+import com.asfoundation.wallet.support.SupportInteractor
+import com.asfoundation.wallet.ui.balance.BalanceInteract
+import com.asfoundation.wallet.ui.gamification.GamificationInteractor
+import com.asfoundation.wallet.wallet_blocked.WalletBlockedInteract
+import io.reactivex.Completable
+import io.reactivex.Observable
+import io.reactivex.Single
+import java.math.BigDecimal
+
+class PaymentMethodsInteract(private val walletService: WalletService,
+                             private val supportInteractor: SupportInteractor,
+                             private val gamificationInteractor: GamificationInteractor,
+                             private val balanceInteract: BalanceInteract,
+                             private val walletBlockedInteract: WalletBlockedInteract,
+                             private val inAppPurchaseInteractor: InAppPurchaseInteractor) {
+
+
+  fun showSupport(gamificationLevel: Int): Completable {
+    return walletService.getWalletAddress()
```

Comment:
Can be converted to an expression body 💯

Reply:
Don't agree